tests/test_message.py

```diff
@@ -44,15 +54,28 @@ def test_constructor(self):
         assert self.message.subject == "SubjectTest"
         assert self.message.body_html == "TextTest"
 
-    def test_ses_send_email_payload(self):
-        """Unit test for `Message.__init__(...)` success."""
-        pass
+    def test_repr(self):
+        """Test the object string representation."""
+        assert self.message.__repr__() == "<Message `test`>"
 
     def test_ses_send_email_json_schema(self):
-        """Unit test for `Message.ses_send_email_payload` success."""
-
-        # use `jsonschema.validate` to assert that payload has proper typing
+        """Use `jsonschema.validate` to ensure that payload has proper typing"""
         validate(
             instance=self.message.ses_send_email_payload,
             schema=self.ses_send_email_schema,
         )
+
+    def test_ses_send_email_payload(self):
+        """Unit test for `Message.ses_send_email_payload` success."""
+        assert self.ses_send_email_payload == self.message.ses_send_email_payload
+
+    def test_sendgrid_send_json_schema(self):
```

Comment:
Should we test inverse case as well?

Reply:
[Optional]

Reply:
Yes, agree.  Will include in later once I add more advanced test cases